nsot/static/src/js/controllers.js

```diff
@@ -506,6 +506,212 @@
 
     });
 
+    app.controller("InterfacesController",
```

Comment:
Ultimately, we should migrate away from $scope and towards the "controller as" notation in prep for Angular2